crates/wasm-pkg-client/src/oci/config.rs

```diff
@@ -32,6 +32,25 @@ impl Clone for OciRegistryConfig {
     }
 }
 
+impl Serialize for OciRegistryConfig {
```

Comment:
This is helpful when you're just converting to another type and immediately serializing: https://serde.rs/container-attrs.html#into

Reply:
Oh that's cool. I've never used that. Let me give it a whirl